src/bazel/karma.conf.js

```diff
@@ -0,0 +1,107 @@
+module.exports = function(config) {
```

Comment:
Add a warning comment about how the testing rule will change? (ppl might come browse this repo to find out what we are planning)